src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,20 @@
+header {
```

Comment:
As we using css modules - don't use styles directly for tag. Always  use classname.